python/nano/tutorial/notebook/inference/tensorflow/tensorflow_inference_bf16.ipynb

```diff
@@ -0,0 +1,269 @@
+      ]
+     },
+     {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+       "from bigdl.nano.tf.keras import InferenceOptimizer\n",
+       "\n",
+       "bf16_model = InferenceOptimizer.quantize(fp32_model, precision='bf16')"
+      ]
+     },
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "> 📝 Note\n",
+       ">\n",
+       "> Please note that the optimization here is in place. That is, the `fp32_model` has also been changed after this `'bf16'` quantization without extra accelerator."
+      ]
+     },
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "## With Extra Accelerator"
+      ]
+     },
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "You could also conduct BF16 mixed precision inference with OpenVINO at the mean time as the accelerator. To achieve this, you could simply **import BigDL-Nano** `InferenceOptimizer`**, and quantize your model with** `accelerator='openvino'`:"
+      ]
+     },
+     {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {},
+      "outputs": [],
+      "source": [
+       "# load the model again if you run the previous code cell \n",
+       "# to conduct bf16 quantization without extra accelerator\n",
+       "fp32_model = keras.applications.MobileNetV2(weights=\"imagenet\")\n",
+       "\n",
+       "from bigdl.nano.tf.keras import InferenceOptimizer\n",
+       "\n",
+       "bf16_ov_model = InferenceOptimizer.quantize(fp32_model, \n",
+       "                                            precision='bf16',\n",
+       "                                            accelerator='openvino')"
+      ]
+     },
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "> 📝 Note\n",
+       ">\n",
+       "> Different from the `'bf16'` quantization without accelerator, the optimization here is not in place.\n",
+       ">\n",
+       "> Please also note that, when you have a custom model to quantize (e.g. inherated from `tf.keras.Model`), you need to specify the `input_spec` parameter to let OpenVINO accelerator know the shape of the model input.\n",
+       ">\n",
+       "> Please refer to [API documentation](https://bigdl.readthedocs.io/en/latest/doc/PythonAPI/Nano/tensorflow.html#bigdl.nano.tf.keras.InferenceOptimizer.quantize) for more information on `InferenceOptimizer.quantize`."
+      ]
+     },
+     {
+      "cell_type": "markdown",
+      "metadata": {},
+      "source": [
+       "After quantizing your model with or without extra accelerator, you could then conduct BF16 mixed precision inferece as normal:"
```

Comment:
A typo `inferece`

Reply:
Thank you for the comments :) I have fixed all of them.